docs/zh_cn/tutorials/how_to_do_regression_test.md

```diff
@@ -0,0 +1,224 @@
+    - [3. 生成的报告](#3-生成的报告)
+        - [模板](#模板)
+        - [示例](#示例)
+    - [4. 支持的后端](#4-支持的后端)
+    - [5. 支持的Codebase及其Metric](#5-支持的Codebase及其Metric)
+    - [6. 注意事项](#7-注意事项)
+    - [7. 常见问题](#8-常见问题)
+
+<!-- TOC -->
+
+## 1. 用法
+
+```shell
+python ./tools/regression_test.py \
+    --deploy-yml "${DEPLOY_YML_PATH}" \
+    --backends "${BACKEND}" \
+    --work-dir "${WORK_DIR}" \
+    --device "${DEVICE}" \
+    --log-level INFO \
+    [--performance]
+```
+
+### 参数解析
+
+- `--deploy-yml` : 需要测试的 codebase，eg.`configs/mmdet/mmdet_regression_test.yaml`，如果设置为 `all` 即全部测试。
+- `--backends` : 筛选测试的后端, 默认 `all`: 测全部`backend`, 也可传入若干个后端，例如 `onnxruntime tesnsorrt`。
+- `--work-dir` : 模型转换、报告生成的路径。
+- `--device` : 使用的设备，默认 `cuda`。
+- `--log-level` : 设置日记的等级，选项包括`'CRITICAL'， 'FATAL'， 'ERROR'， 'WARN'， 'WARNING'， 'INFO'， 'DEBUG'， 'NOTSET'`。默认是`INFO`。
+- `--performance` : 是否测试精度，加上则测试转换+精度，不加上则只测试转换
+
+## 例子
+
+1. 测试 mmdet 和 mmpose 的所有 backend 的 转换+精度
+
+```shell
+python ./tools/regression_test.py \
+    --deploy-yml ./configs/mmdet/mmdet_regression_test.yaml ./configs/mmpose/mmpose_regression_test.yaml \
+    --backends all \
+    --work-dir "../mmdeploy_regression_working_dir" \
+    --device "cuda" \
+    --log-level INFO \
+    --performance
+```
+
+2. 测试 mmdet 和 mmpose 的某几个 backend 的 转换+精度
+
+```shell
+python ./tools/regression_test.py \
+    --deploy-yml ./configs/mmdet/mmdet_regression_test.yaml ./configs/mmdet/mmpose.yaml \
+    --backends onnxruntime tesnsorrt \
+    --work-dir "../mmdeploy_regression_working_dir" \
+    --device "cuda" \
+    --log-level INFO \
+    --performance
+```
+
+3. 测试 mmdet 和 mmpose 的某几个 backend，只需测试转换
+
+```shell
+python ./tools/regression_test.py \
+    --deploy-yml ./configs/mmdet/mmdet_regression_test.yaml ./configs/mmdet/mmpose.yaml \
+    --backends onnxruntime tesnsorrt \
+    --work-dir "../mmdeploy_regression_working_dir" \
+    --device "cuda" \
+    --log-level INFO
+```
+
+## 2. 回归测试配置文件
+
+### 示例及参数解析
+
+```yaml
+globals:
+  codebase_name: mmocr # 回归测试的 codebase 名称
+  codebase_dir: ../mmocr # 回归测试的 codebase 路径
+  checkpoint_force_download: False # 回归测试是否重新下载模型即使其已经存在
+  checkpoint_dir: ../mmdeploy_checkpoints # 回归测试是否下载模型的路径
+  images: # 测试使用图片
+    img_224x224: &img_224x224 ./tests/data/tiger.jpeg
+    img_300x300: &img_300x300
+    img_800x1344: &img_cityscapes_800x1344
+    img_blank: &img_blank
+  metric_info: &metric_info # 指标参数
+    hmean-iou: # 命名根据 metafile.Results.Metrics
+      eval_name: hmean-iou # 命名根据 test.py --metrics args 入参名称
+      metric_key: 0_hmean-iou:hmean # 命名根据 eval 写入 log 的 key name
+      tolerance: 0.1 # 容忍的阈值区间
+      task_name: Text Detection # 命名根据模型 metafile.Results.Task
+      dataset: ICDAR2015 #命名根据模型 metafile.Results.Dataset
+    word_acc: # 同上
+      eval_name: acc
+      metric_key: 0_word_acc_ignore_case
+      tolerance: 0.2
+      task_name: Text Recognition
+      dataset: IIIT5K
+  convert_image: &convert_image # 转换会使用到的图片
+    input_img: *img_224x224
```

Comment:
Do we support non-image inputs? Such as point cloud in mmdetection3d.

Reply:
Theoretically, it is supported. This parameter will be passed to arg named `img` when using `./tools/deploy.py` to convert the model to specifiy backend in the script